extensions/cloud/portability-cloud-local/build.gradle

```diff
@@ -17,4 +17,8 @@
 dependencies {
     compile project(':portability-spi-cloud')
     compile project(':portability-spi-transfer')
+
```

Comment:
nit: consider moving this to a separate PR since it's not related to the twitter integration